eox_tagging/api/v1/filters.py

```diff
@@ -52,10 +45,35 @@ def filter_by_target_object(self, queryset, name, value):
         return queryset
 
     def filter_target_types(self, queryset, name, value):  # pylint: disable=unused-argument
-        """Filter that returns targets by their type."""
+        """
+        Filter that returns targets using their type.
+
+        **SPECIAL CASE**: course enrollments.
+
+        If the user wants to filter by target_type courseenrollment and wants to add filters on
+        user or course, it must pass the following:
+
+            - target_type: if the other arguments are passed this is used to differentiate between
+            course_id from courseoverview and username from user object.
+            - enrollment_course_id (optional)
+            - enrollment_username (optional)
+        """
+        target_id = {}
+
+        if value == "courseenrollment":
+            course_id = self.request.query_params.get("enrollment_course_id")
+            username = self.request.query_params.get("enrollment_username")
+            target_id.update({"username": username, "course_id": course_id})
+
+        if any(object_id for _, object_id in target_id.items()):
```

Comment:
```suggestion
        if any(object_id for object_id in target_id.values()):
```
Would be better, no?

Reply:
Yes! Thanks for the suggestion.